test/support/fixtures/accounts_fixtures.ex

```diff
@@ -0,0 +1,25 @@
+defmodule GoogleSearchDataViewer.AccountsFixtures do
```

Comment:
In case we already use the `GoogleSearchDataViewer.Factory`, I think we also can use it to create data for testing, for example:

```elixir
insert(:user)
```

More functions: https://hexdocs.pm/ex_machina/ExMachina.Ecto.html

We can remove the fixture here.

Reply:
Yes, done this and at the same time moved the password hashing function outside of the schema so it can be used in the factory and tests. Thanks